### DISCUSSION TITLE
Request to Update gRPC dependeny from 1.44.1 to 1.46.0

### DISCUSSION THREAD

**Discussion by jaira** [firebase/firebase-android-sdk#3758] [Q&A] [answered]
Hi Firebase Developers, 

I would like to ask,  is it possible to update the gRPC dependencies of Firestore from 1.44.1 to 1.46.0 asap?
This request was flagged because a util class in gRPC 1.44.1 has raised vulnerability concerns regarding some hashing methods for MD5 and SHA1, which are considered weak hashing algorithms. I have already contacted one of the developer of gRPC and he included the code cleanup in v1.46.0. Here is the Util.java class with cleanup that needs to be updated in Firestore dependencies
https://github.com/grpc/grpc-java/blob/master/okhttp/third_party/okhttp/main/java/io/grpc/okhttp/internal/Util.java

Hoping faster response regarding this matter since this is causing delay for our code deployment with our clients.

Thanks.

**Comment by rlazo** [comment, marked as answer]
Our current gRPC dep version is to 1.57.2
